cnf-certification-test/lifecycle/suite.go

```diff
@@ -555,3 +565,23 @@ func TestPodTolerationBypass(env *provider.TestEnvironment) {
 
 	testhelper.AddTestResultLog("Non-compliant", podsWithRestrictedTolerationsNotDefault, tnf.ClaimFilePrintf, ginkgo.Fail)
 }
+func testStorageRequiredPods(env *provider.TestEnvironment) {
+	oc := clientsholder.GetClientsHolder()
+	var nonCompliantPods []*provider.Pod
+	for _, pod := range env.Pods {
+		for _, podOwner := range pod.ObjectMeta.GetOwnerReferences() {
+			if podOwner.Kind == statefulSet {
```

Comment:
There are too much inner levels, I'd do:
```
if podOwner.Kind != statefulSet {
	continue
}
statefulset, err := oc.K8sClient.AppsV1().StatefulSets(pod.Namespace).Get(context.TODO(), podOwner.Name, apiv1.GetOptions{})
```

Reply:
If it is Not equal to "statefulSet" then we will continue because the condition is not met and it will continue to the next podOwner, but I agree that it is clearer in your way. So I will change it.